### ISSUE TITLE
Issue with prefersLargeTitles when scrollview bounce locked.

### ISSUE THREAD

**Issue by DhavalBhimani** [SCENEE/FloatingPanel#304]
### Short description
When i set UINavigationController controller as contentViewController and use prefersLargeTitles for rootviewcontroller of it. i am not able to get title back to its position when scroll down the floatingpanelcontroller. and title sticks to the top. Is there any solution for this in FloatingPanel

Thanks.

![ezgif-2-4f403aed2ea3](https://user-images.githubusercontent.com/29978287/71877951-7814df80-3150-11ea-9cd7-86accff3bf4f.gif)

### Expected behavior
- title should go back to large title when scrolled down.
### Actual behavior
- It breaks the title UI. some time it goes back and some times it stays on top.

**Code example that reproduces the issue**
`let nav = UINavigationController(rootViewController: self.nearByVC)
			self.floatingPanelController.set(contentViewController: nav)
			self.floatingPanelController.track(scrollView: self.nearByVC.tableView)
			self.floatingPanelController.isRemovalInteractionEnabled = false
			self.floatingPanelController.surfaceView.contentInsets = .init(top: 20, left: 0, bottom: 0, right: 0)`
### Environment
Debug
**Library version**
1.71
**Installation method**
pod 'FloatingPanel'

**iOS version(s)**
13.3
**Xcode version**
11.3

**Comment by SCENEE** [comment]
Thank you for your report. I've acknowledged this issue. I'm going to try to fix this issue.

**Comment by jsm174** [comment]
I'm having a similar issue. 

I have the following stack:

```
TabBarController
   (Selected Tab)
      NavigationController
         ViewController
             FloatingPanel
                 NavigationController
                     TableViewController
```

When I set the `UINavigationController` to `prefersLargeTitles`, the minute I start scrolling the `tableView`, it shrinks about the height of large titles. (From the pic, the bottom of the scroll indicator is the bottom of the `UITableViewController`) 

![floating](https://user-images.githubusercontent.com/1197137/73566910-1a17a700-4433-11ea-91d6-fce4e1c2edff.png)

**Comment by jsm174** [comment]
So I played with this some more.

When the fcp contains a navigation controller, and your using that fcp in a tab of a `uitabviewcontroller`, I needed to subtract the height of the tab bar from the surfaceView's content insents.

```
fpc.surfaceView.contentInsets = .init(top: 0, left: 0, bottom: -83, right: 0)
```

**Comment by SCENEE** [comment]
Thanks you @jsm174. Yeah I've confirmed it's related to the content insets. 
I figured out that a scroll view in a nav bar displaying a large title needs to consider its safe area inset to calculate the visual offset zero.
Therefore I'm planning to add a new delegate method to control scrolling a content in a tracking scroll view against the panel's pan gesture.

**Comment by SCENEE** [comment]
Please use `floatingPanel(_:contentOffsetForPinning:)` to achieve this expected behavior. This new delegate method will be released in v1.7.3. [Here] is an example. Thank you.
